flyio/caveats.md

```diff
@@ -408,3 +404,97 @@ access request does not specify a command.
     ]
   }
 ```
+
+
+## Discharge Access Control
+
+There are additional access controls that can be performed when checking
+third party discharges issued by Fly services.
+The following access request structure is used when checking
+if an access is allowed. It carries information about the users the discharge
+token can be used by:
+
+```
+type DischargeRequest struct {
```

Comment:
This is only really relevant to the https://auth.fly.io 3p. No harm in documenting it, but I don't think anyone external will ever interact with it.